thoth/storages/graph/models_performance.py

```diff
@@ -80,9 +80,9 @@ def create_from_report(
     ) -> "PerformanceIndicatorBase":
         """Create performance indicator record together with related observed performance edge based on inspection."""
         # Place core parts of the base class into the model.
-        framework = inspection_document["job_log"]["stdout"].get("framework")
-        if not framework:
-            _LOGGER.warning("No machine learning framework specified in performance indicator %r", cls.__name__)
+        library = inspection_document["job_log"]["stdout"].get("library")
+        if not library:
+            _LOGGER.warning("No library specified in performance indicator %r", cls.__name__)
```

Comment:
Do we want to sync such results? I'm wondering about their usefulness.

Reply:
This is required to distinguish between `tensorflow` or `pytorch`, now we have `pybench`, these are python modules at the end. So I thought library might be good.

Reply:
I was pointing to a case when it is not set. It might be good idea to refuse results that do not have `library` (in this case) set - we will not use them anyway as they are not bound to any library.

If we will just perform performance indicator tests, then I think we shouldn't trigger graph sync (this logic would be in Argo workflows).

Reply:
But the graph-sync would tell us if that key is missing, we can make it mandatory in the schema (nullable=False) so if that is not provided, it fails. So we don't need:

```
library = inspection_document["job_log"]["stdout"].get("library")
        if not library:
            _LOGGER.warning("No library specified in performance indicator %r", cls.__name__)
```